src/main/java/spoon/compiler/Environment.java

```diff
@@ -43,6 +43,10 @@
 	 */
 	void setComplianceLevel(int level);
 
+	TO_STRING_MODE getToStringMode();
```

Comment:
I suggest to rename `toString mode` to `pretty printing mode`, because CtElement#toString will always use VANILA model independent on that setting. 
In other words: setToStringMode actually doesn't influence behavior of CtElement#toString, but it influence how pretty printing works.
WDYT?